src/profile/voyages/tag/vsca.js

```diff
@@ -1,119 +1,322 @@
+                    b = a.createElement("script");
+                    b.language = "javascript";
+                    b.type = "text/javascript";
+                    b.async = 1;
+                    b.charset = "utf-8";
+                    for (l in utag.loader.GV(o.attrs)) {
+                        b[l] = o.attrs[l];
+                    }
+                    b.src = o.src;
+                }
+                if (o.id) {
+                    b.id = o.id
+                };
+                if (typeof o.cb == "function") {
+                    if (b.addEventListener) {
+                        b.addEventListener("load", function() {
+                            o.cb()
+                        }, false);
+                    } else { /* old IE support */
+                        b.onreadystatechange = function() {
+                            if (this.readyState == 'complete' || this.readyState == 'loaded') {
+                                this.onreadystatechange = null;
+                                o.cb()
+                            }
+                        };
+                    }
+                }
+                l = o.loc || "head";
+                c = a.getElementsByTagName(l)[0];
+                if (c) {
+                    utag.DB("Attach to " + l + ": " + o.src);
+                    if (l == "script") {
+                        c.parentNode.insertBefore(b, c);
+                    } else {
+                        c.appendChild(b)
+                    }
+                }
+            }
+        } else {
+            u.loader = utag.ut.loader;
+        }
+        // End Tealium loader
+        // Start encode
+        u.encode = function(a, b) {
+            b = "";
+            try {
+                b = encodeURIComponent(a)
+            } catch (e) {
+                utag.DB(e);
+            }
+            if (b == "") {
+                b = escape(a);
+            }
+            return b
+        }
+        // End encode
 
-        c = [];
+        u.ev = {
+            "view": 1
+        };
 
-        for (d in utag.loader.GV(u.map)) {
-          if (typeof b[d] !== "undefined" && b[d] !== "") {
-            e = u.map[d].split(",");
-            for (f = 0; f < e.length; f++) {
-              if (!u.data.hasOwnProperty(e[f])) {
-                c.push(e[f] + "##kvp_delim##" + u.encode(b[d]));
-              }
-              u.data[e[f]] = b[d];
+        // Function to replace @@ placeholders
+        u.rp = function(a, b) {
+            if (typeof a !== "undefined" && a.indexOf("@@") > 0) {
+                a = a.replace(/@@([^@]+)@@/g, function(m, d) {
+                    if (b[d]) {
+                        return u.encode(b[d]);
+                    } else {
+                        return "";
+                    }
+                });
             }
-          }
-        }
+            return a;
+        };
 
-        u.data.secure_base_url = u.data.secure_base_url || u.data.base_url;
+        ##
```

Comment:
Can you undo the formatting for the non-custom portion of the code? That way we can verify that the non-custom portion remains untouched when I update the profile in Tealium. One of the things that the formatter would break is this line with ##UTGEN##. Tealium will attempt to replace the pattern when generating the final code.